app/(admin)/openmailer/campaigns/[id]/CampaignEditForm.tsx

```
@@ -0,0 +1,253 @@
+          <p className="mt-1 font-medium text-gray-900">{campaign.listName}</p>
+        </div>
+      </>
+    );
+  }
+
+  if (isEditing) {
+    return (
+      <div className="space-y-6 rounded-lg border border-gray-200 bg-white p-6 shadow-sm">
+        <h2 className="text-lg font-semibold text-gray-900">Edit campaign</h2>
+        <div>
+          <label className="mb-2 block text-sm font-medium text-gray-700">
```

ESLint

[error] A form label must be associated with a control.

(jsx-a11y/label-has-associated-control)